cmd/snap/cmd_model.go

```diff
@@ -304,6 +310,75 @@ func (x *cmdModel) Execute(args []string) error {
 					fmt.Fprintln(w, "device-key-sha3-384: |")
 					wrapLine(w, []rune(headerString), "  ", termWidth)
 				}
+			case "snaps":
+				// also flush the writer before continuing so the previous keys
+				// don't try to align with this key
+				w.Flush()
+				snapsHeader, ok := headerValue.([]interface{})
+				if !ok {
+					return invalidTypeErr
+				}
+				if len(snapsHeader) == 0 {
+					// unexpected why this is an empty list, but just ignore for
+					// now
+					continue
+				}
+				fmt.Fprintf(w, "snaps:\n")
+				for _, sn := range snapsHeader {
+					snMap, ok := sn.(map[string]interface{})
+					if !ok {
+						return invalidTypeErr
+					}
+					// iterate over all keys in the map in a stable, visually
+					// appealing ordering
+					// first do snap name, which will always be present
+					name, ok := snMap["name"]
+					if !ok {
+						// TODO: more specific error ?
```

Comment:
doesn't matter much as we parse the assertion and that will fail anyway if name is missing, so this is just extra paranoia

Reply:
Maybe just a comment static that assertion was parsed before and found to be valid (where the snap name is required)?

Reply:
I added a comment and skipped some of this paranoia